### PR TITLE
Update how-can-i-fix-browser-was-not-found.md

### PR DESCRIPTION
**Pull request by BaronSam3di**
Based on the conversation [here](https://groups.google.com/g/zaproxy-users/c/52GMZl6f6jY/m/E2QDgnwOAwAJ) I have elaborated on browser updates and tweaked a few grammars I saw 😌

### CODE REVIEW

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 1]
```diff
@@ -11,12 +11,15 @@ you would otherwise get from the ZAP root CA certificate.
 
 But what can you do if ZAP fails to launch your browser?
 
-### Check for Updates
+### Check for Updates
```
Comment (thc202) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:14]: This should be reverted.

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 2]
```diff
@@ -11,12 +11,15 @@ you would otherwise get from the ZAP root CA certificate.
...
 But what can you do if ZAP fails to launch your browser?
 
-### Check for Updates
+### Check for Updates 
 
-The first thing to do is to 'Check for Updates' and install any add-ons which are out of date, 
-in particular the webdriver and [Selenium](/docs/desktop/addons/selenium/) ones.
-Browsers are updated very frequently and the updates often require corresponding changes to the webdrivers.
-If you dont have the right webdrivers then ZAP will not be able to launch the browsers.
+In Zap:
```
Comment (thc202) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:16]: `ZAP`

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 3]
```diff
@@ -11,12 +11,15 @@ you would otherwise get from the ZAP root CA certificate.
...
 
-### Check for Updates
+### Check for Updates 
 
-The first thing to do is to 'Check for Updates' and install any add-ons which are out of date, 
-in particular the webdriver and [Selenium](/docs/desktop/addons/selenium/) ones.
-Browsers are updated very frequently and the updates often require corresponding changes to the webdrivers.
-If you dont have the right webdrivers then ZAP will not be able to launch the browsers.
+In Zap: 
+- The first thing to do is to 'Check for Updates' in Zap and install any add-ons which are out of date,
```
Comment (thc202) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:17]: Same here.

File: site/content/faq/how-can-i-fix-browser-was-not-found.md [hunk 4]
```diff
@@ -11,12 +11,15 @@ you would otherwise get from the ZAP root CA certificate.
...
-in particular the webdriver and [Selenium](/docs/desktop/addons/selenium/) ones.
-Browsers are updated very frequently and the updates often require corresponding changes to the webdrivers.
-If you dont have the right webdrivers then ZAP will not be able to launch the browsers.
+In Zap: 
+- The first thing to do is to 'Check for Updates' in Zap and install any add-ons which are out of date, 
+in particular, the webdriver and [Selenium](/docs/desktop/addons/selenium/) ones.
+
+In your local browser:
+- Browsers are updated very frequently, and the updates often require corresponding changes to the webdrivers.
+If you dont have the right webdrivers then ZAP will not be able to launch the browsers. **Zap ties to your local browser**, so even though it launches its own instance of web browsers, check you have the latest version installed locally.
```
Comment (kingthorin) [inline site/content/faq/how-can-i-fix-browser-was-not-found.md:22]:
```suggestion
If you don’t have the right webdrivers then ZAP will not be able to launch the browsers. **ZAP ties to your local browser**, so even though it launches its own instance of web browsers, check you have the latest version installed locally.
```

**Review by kingthorin — DISMISSED** [review]

**Comment by kingthorin** [comment]
Oops that shouldn’t be approve….

**Comment by thc202** [comment]
@BaronSam3di ping.

**Comment by thc202** [comment]
Comments addressed.

Thank you!

**Review by psiinon — APPROVED** [review]

**Review by kingthorin — APPROVED** [review]

**Pull request merged**